.vale/styles/Konflux/DirectAddress.yml

```
@@ -0,0 +1,9 @@
+extends: substitution
+message: "Use 'you' instead of '%s' for direct address"
```

probably this should be the fix
```suggestion
message: "Use '%s' instead of '%s' for direct address"
```